db/python/tables/sample.py

```diff
@@ -97,6 +97,80 @@ async def update_sample(
         await self.connection.execute(_query, {**values, 'id': id_})
         return id_
 
+    async def merge_samples(
+        self,
+        id_keep: int = None,
+        id_merge: int = None,
+        author: str = None,
+    ):
+        """Merge two samples together"""
+        sid_merge = sample_id_format(id_merge)
+        _, samples = await self.get_samples_by(sample_ids=[id_keep, id_merge])
```

Comment:
This was probably bad advice, looks like this returns a SampleModel which is casting the ID to a string, causing errors, I think your original code here was better, but wrapping it in a asyncio.gather so we don't have to wait for 2 sequential round trips:

```suggestion
                (_, sample_keep), (_, sample_merge) = await asyncio.gather(
            self.get_single_by_id(id_keep),
            self.get_single_by_id(id_merge),
        )
```